source/reference/minio-server/settings/console.rst

```diff
@@ -176,6 +178,148 @@ Log Query URL
       This setting does not have a configuration variable setting.
       Use the Environment Variable instead.
 
+Content Security Policy
+~~~~~~~~~~~~~~~~~~~~~~~
+
+*Optional*
+
+Set the `Content-Security-Policy <https://en.wikipedia.org/wiki/Content_Security_Policy>`__ header in MinIO Console responses.
+Defaults to ``default-src 'self' 'unsafe-eval' 'unsafe-inline';``
+
+.. tab-set::
+
+   .. tab-item:: Environment Variable
+      :sync: envvar
+
+      .. envvar:: MINIO_BROWSER_CONTENT_SECURITY_POLICY
+
+      .. code-block:: shell
+         :class: copyable
+
+         set MINIO_BROWSER_CONTENT_SECURITY_POLICY="default-src 'self' 'unsafe-eval' 'unsafe-inline';"
+
+   .. tab-item:: Configuration Setting
+      :sync: config
+
+      .. mc-conf:: browser csp_policy
+         :delimiter: " "
+
+      .. code-block:: shell
+
+         mc admin config set browser \
+            csp_policy="default-src 'self' 'unsafe-eval' 'unsafe-inline';" \
+            [ARGUMENT=VALUE ...]
+
+
+Strict Transport Security
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+*Optional*
+
+Set the `Strict-Transport-Security <https://en.wikipedia.org/wiki/HTTP_Strict_Transport_Security>`__ header in MinIO Console responses.
+Set the appropriate environment variable or configuration settings according to the HSTS requirements for your environment.
+
+.. tab-set::
+
+   .. tab-item:: Environment Variables
+      :sync: envvar
+
+      .. envvar:: MINIO_BROWSER_HSTS_INCLUDE_SUB_DOMAINS
+
+         Set to ``on`` to append ``includeSubDomains`` to the ``Strict-Transport-Security`` header.
+	 Defaults to ``off``.
+
+      .. code-block:: shell
+         :class: copyable
+
+         set MINIO_BROWSER_HSTS_INCLUDE_SUB_DOMAINS="on"
+
+
+      .. envvar:: MINIO_BROWSER_HSTS_PRELOAD
+
+	 Set to ``on`` to include this header in the HSTS preloading list.
+	 Defaults to ``off``.
+
+      .. code-block:: shell
+         :class: copyable
+
+         set MINIO_BROWSER_HSTS_INCLUDE_SUB_DOMAINS="on"
+
+      .. envvar:: MINIO_BROWSER_HSTS_SECONDS
+
+	 The ``max_age`` the configured policy remains in effect, in seconds. 
+	 Defaults to ``0``.
```

Comment:
is ``0`` here the equivalent of indefinite? Otherwise these two sentences combined implies the policy expires immediately.

Reply:
"0" is default value, it means "expiration is in 0 seconds" which is the same that not render the header at all, in other words, "0" is disabled.